### ISSUE TITLE
Freetype includes are inconsistent.

### ISSUE THREAD

**Issue by vlj** [Microsoft/vcpkg#133]
When trying to build harfbuzz with freetype support I found that the include path to freetype is inconsistent with the headers it contains : for instance ft2build.h has an "#include <freetype/config/ftheader.h>" directive but freetype include uses freetype2 directory name ; the file should use the "freetype2/config/ftheader.h" path.
The freetype2/config/ftheader also declares the freetype/ path instead of the freetype2/ as include.

**Comment by aybassiouny** [comment]
I faced same issue while trying to port ogre.

**Comment by ras0219-msft** [comment]
Ok, I believe I've fixed this with c494892.

It's a bit unclear what the "standard" layout is.
- Debian uses `freetype2/config/ftheader.h` (https://packages.debian.org/jessie/amd64/libfreetype6-dev/filelist)
- Archlinux uses `freetype2/freetype/config/ftheader.h` (https://www.archlinux.org/packages/extra/x86_64/freetype2/)
- The files deployed by freetype itself appear to reference `freetype/config/ftheader.h`, but are laid out as `freetype2/freetype/config/ftheader.h`

I went with removing the `freetype2` directory, so now the layout is
```
include\ft2build.h
include\freetype\config\ftheader.h
include\freetype\ft*.h
```
This appears to comply best with the included example in the header files.
